examples/Console/TestConsoleExporter.cs

```diff
@@ -41,14 +41,14 @@ internal static object Run(ConsoleOptions options)
             // which decide to use Open Telemetry.
 
             // Libraries would simply write the following lines of code to
-            // emit activities, which are the .NET representation of OT Spans.
+            // emit activities, which are the .NET representation of OTel Spans.
```

Comment:
For comments, I think we probably want to use the full name given we already use the full name in the copyright comment `// Copyright The OpenTelemetry Authors`.

Reply:
OK!